tests/core/vm/test_computation.py

```diff
@@ -0,0 +1,184 @@
+import pytest
+
+from evm.vm import (
+    Computation,
+    Message,
+)
+
+from eth_utils import (
+    to_canonical_address,
+)
+
+from evm.exceptions import (
+    ValidationError,
+)
+
+CANONICAL_ADDRESS_1 = to_canonical_address("0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6")
+CANONICAL_ADDRESS_2 = to_canonical_address("0xcd1722f3947def4cf144679da39c4c32bdc35681")
+
+
+@pytest.fixture
+def message():
+    message = Message(
+        origin=to_canonical_address(CANONICAL_ADDRESS_2),
+        to=CANONICAL_ADDRESS_1,
+        sender=to_canonical_address(CANONICAL_ADDRESS_2),
+        value=100,
+        data=b"0x",
+        code=b"0x",
+        gas=100,
+        gas_price=1,
+    )
+    return message
+
+
+@pytest.fixture
+def computation(message):
+    computation = Computation(
+        vm=1,
+        message=message
+    )
+    return computation
+
+
+@pytest.fixture
+def create_error():
+    def computation_error(computation):
+        computation.gas_meter.gas_remaining = 0
```

Comment:
@pipermerriam Would it be okay if I made an error with `computation.extend_memory(32, 100000)`?  (causing it to actually run out of gas)